test/util/cluster/cluster.go

```diff
@@ -331,3 +332,10 @@ func GetNodeSecureBootState(clients *testclient.ClientSet, nodeName, namespace s
 
 	return strings.Contains(stdout, "[integrity]") || strings.Contains(stdout, "[confidentiality]"), nil
 }
+
+func VirtualCluster() bool {
+	if v, exist := os.LookupEnv("VIRTUAL_ENV"); exist && v != "" {
```

Comment:
I'd choose a more specific name, as "virtual" is a spoiled word. Something like `CLUSTER_HAS_EMULATED_PF` or `CLUSTER_HAS_IGB_NIC`, as it's used to see if the emulated nic is present on the cluster, right?

Reply:
using `CLUSTER_HAS_EMULATED_PF` as IGB is a valid real nic from intel not only virtual one